app/src/main/java/com/github/libretube/ui/sheets/VideoOptionsBottomSheet.kt

```diff
@@ -120,7 +119,9 @@ class VideoOptionsBottomSheet(
                         // add video to watch history
                         runCatching {
                             RetrofitInstance.api.getStreams(videoId)
-                        }.getOrNull()?.let { DatabaseHelper.addToWatchHistory(videoId, it) }
+                        }.getOrNull()?.let {
+                            DatabaseHelper.addToWatchHistory(videoId, it)
+                        }
```

Comment:
This could potentially also use `StreamItem`, but it only saves a localized version of the `uploadDate` with no way of reversing it to a `LocalDate`.

Reply:
It's not localized, it's just a string that is used for storing it. We can use the `uploaded` parameter of `StreamItem` and convert it to a `LocalDate`, since `uploaded` represents the milliseconds of the date.
```kotlin
val uploadDate = Instant.ofEpochMilli(epoch).atZone(ZoneId.systemDefault()).toLocalDate()
```

Reply:
Hmm, when I tried to use parsed it to a `LocalDate`, it failed, because 23 hours ago is not a valid date.